src/helpers/relative-activity.tsx

```diff
@@ -30,26 +30,25 @@ export function queryRelativeActivity(startDate: Date, endDate: Date, dataTypes:
 
     return Promise.allSettled(promises).then((results) => {
         dataTypes.forEach((dataType, index) => {
-            relativeActivityResults[dataType.dailyDataType] = {};
```

Comment:
subtle change here to this - I think here the correct behavior wasn't to return an empty set of results for a data type if there is an error...but rather we should not add that data type key to the result dictionary at all.  Doesn't matter a ton but the micro trend treats an empty result as having "some data" so it was rendering empty components in some cases that should just be hidden